parsec/core/gui/workspace_button.py

```diff
@@ -122,7 +122,7 @@ def __init__(
 
     @property
     def is_shared(self):
-        return len(self.users_roles) > 1
+        return sum(1 for _, u in self.users_roles.values() if u.revoked_on is None) > 1
```

Comment:
Would be good to add a comment here to underline we need this only to filter out users that have been revoked since the last workspace re-encryption